test/e2e/framework/pod/create.go

```diff
@@ -249,9 +249,6 @@ func MakePodSpec(podConfig *Config) *v1.PodSpec {
 	podSpec.Containers[0].VolumeMounts = volumeMounts
 	podSpec.Containers[0].VolumeDevices = volumeDevices
 	podSpec.Volumes = volumes
-	if runtime.GOOS != "windows" {
```

Comment:
from https://github.com/kubernetes/kubernetes/pull/99888/files#r596637806

Reply:
could you make a separate PR for this fix? want to make sure this can be merged sooner.

Reply:
Opened https://github.com/kubernetes/kubernetes/pull/100540